#!/usr/bin/python

PACKAGE="spline_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("ds", double_t, 0, "Waypoints sampling interval along the path.", 0.2, 0.1, 1.0)
gen.add("trajectory_length", double_t, 0, "Length of trajectory the planner generates", 15.0, 1.0, 300.0)
gen.add("max_linear_speed", double_t, 0, "Maximum allowed linear speed.", 6.0, 0.1, 32.0)
gen.add("max_total_acceleration", double_t, 0, "Maximum allowed total acceleration.", 3.0, 0.1, 10.0)

exit(gen.generate(PACKAGE, "Planner", "Planner"))
